spec/services/edge_cache/bust_article_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.describe EdgeCache::BustArticle, type: :service do
+  let(:article) { create(:article) }
+
+  before do
+    allow(article).to receive(:purge)
+    allow(described_class).to receive(:bust)
+  end
+
+  it "busts the cache" do
```

Comment:
I'm open to ideas on how to expand this test coverage!

Reply:
Some ideas:
- we could test that `bust` was called _x_ number of times
- we could test that the organization slug is busted if it has an org
- we could test that `bust_home_pages ` and `bust_tag_pages` [were called](https://github.com/forem/forem/pull/11788/files#diff-f9b1966dac6d323e3c03dfd9e957e73a5e586df05546904669c677d14b9b93ccR17-R18)?

i don't think all of that is probably necessary, but it would make it really clear to us if the service was changed down the road because the tests would probably catch it 🤷

Reply:
Added all 3 suggestions 💁🏽‍♂️